cartridges/int_externalCustomerData/cartridge/scripts/initCustomerService.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var LocalServiceRegistry = require('dw/svc/LocalServiceRegistry');
+
+function getToken() {
+    var getTokenService = LocalServiceRegistry.createService("http.customer.service", {
+        createRequest: function (svc, params) {
+            svc.setRequestMethod("POST");
+            svc.setURL(params.URL);
+            svc.addHeader('Content-Type', 'application/json');
+            return JSON.stringify({
+                email: svc.configuration.credential.user,
+                password: svc.configuration.credential.password
+            });
+        },
+        parseResponse: function (svc, response) {
+            var data = response;
+            return response;
+        },
+    });
+    return getTokenService;
+}
+
+function requestCustomerDataToExternalService() {
+    var tokenStorage = require('./tokenStorage');
+    var CustomerService = LocalServiceRegistry.createService("http.customer.service", {
+        createRequest: function (svc, params) {
```

Comment:
Need to commit metadata for the service definition. Also I don't see formatting of the cartridge to be prepared as LINK cartridge

Reply:
done